src/xgov-dapp/src/features/vote/VoteResults.tsx

```diff
@@ -145,23 +204,73 @@ export const VoteResults = ({
             </>
           ))}
         {!isLoadingVotingRoundResults &&
-          votingRoundMetadataClone?.questions.map((question, index) => (
-            <div key={question.id}>
-              {question.metadata && (
-                <ProposalCard
-                  title={question.prompt}
-                  description={question.description}
-                  category={question.metadata.category}
-                  focus_area={question.metadata.focus_area}
-                  link={question.metadata.link}
-                  threshold={question.metadata.threshold}
-                  ask={question.metadata.ask}
-                  votesTally={votingRoundResults && votingRoundResults[index] ? votingRoundResults[index].count : 0}
-                  hasClosed={true}
-                />
-              )}
+          votingRoundMetadataClone?.questions
+            .filter((q) => !isReserveList(q))
+            .map((question) => (
+              <div key={question.id}>
+                {question.metadata && (
+                  <ProposalCard
+                    title={question.prompt}
+                    description={question.description}
+                    category={question.metadata.category}
+                    focus_area={question.metadata.focus_area}
+                    link={question.metadata.link}
+                    threshold={question.metadata.threshold}
+                    ask={question.metadata.ask}
+                    votesTally={
+                      oIdsToCounts && question.options.length > 0 && oIdsToCounts[question.options[0].id]
+                        ? oIdsToCounts[question.options[0].id]
+                        : 0
+                    }
+                    hasClosed={true}
+                  />
+                )}
+              </div>
+            ))}
+        {reserveList && reserveList.length > 0 && (
+          <>
+            <div className="col-span-1 xl:col-span-3">
+              <Typography variant="h4">Reserve List</Typography>
             </div>
-          ))}
+            {isLoadingVotingRoundData ||
```

Comment:
Add `isReserveListEnabled` to prevent reserve list skeleton from rendering on v1

Reply:
taken care by `reserveList && reserveList.length > 0`